include/swift/AST/DiagnosticsSema.def

```diff
@@ -1005,6 +1005,11 @@ ERROR(module_not_testable,Fatal,
 ERROR(module_not_compiled_for_private_import,none,
       "module %0 was not compiled for private import", (Identifier))
 
+ERROR(package_module_not_compiled_from_source,Fatal,
+      "module %0 is a package module but was built from interface; "
+      "it must be compiled from source instead",
```

Comment:
Nit: perhaps: 
`module %0 is a package module but was built from interface; package modules can only be loaded when built from source`?

`it must be compiled from source instead` sounds prescriptive, when I don't see exactly what the user is to do about it.

Reply:
sounds much better; will update